### PR TITLE
feat: Blazor sample

### PR DESCRIPTION
**Pull request by felixindynamsoft**
Created Blazor Sample for DBR JS with DBR bundle

Additional reference: https://github.com/dynamsoft-rd-0/dlrjs-mass-samples/tree/main/BlazorMRZ

### CODE REVIEW

**Comment by Keillion** [comment]
1) How about removing useless files that generated automatically, like weather, counter.

2) When dealing with cross-language calls, we have to be careful with large binary arrays. Some unnecessary data transfers will affect performance.
   In image decoding,  we get binary to dotNet, send back to js, then wrap as Blob, then unwrap Blob in our SDK inner logic...this is expensive.
   We can let js can get inputElement by id or className, `inputElement.files` to get blobs.

**Comment by felixindynamsoft** [comment]
@Keillion Updated. thank you!

File: hello-world/blazor/README.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    const inputElement = document.getElementById("inputElement");
+    const resultsContainer = document.getElementById("results");
+
+    const file = inputElement.files[0]; // Get the first file from the input element
+
+    try {
+        if (file) {
+            // Convert file to blob
+            const arrayBuffer = await file.arrayBuffer();
+            const blob = new Blob([arrayBuffer]);
```
Comment (Keillion) [inline hello-world/blazor/README.md:244]: `file.instanceOf(Blob) === true`, so do not need to convert.

Comment (Keillion) [reply hello-world/blazor/README.md:244]: After get the file, you can reset `inputElement.value`, so user can test a image repeatedly.

File: hello-world/blazor/README.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    // reference: https://learn.microsoft.com/en-us/aspnet/core/blazor/javascript-interoperability/call-javascript-from-dotnet?view=aspnetcore-8.0#invoke-js-functions
+
+    private async Task DecodeImageTask(InputFileChangeEventArgs e)
+    {
+        // Dynamically load decode image module
+        await JS.InvokeVoidAsync("loadDecodeImageModule");
+
+        var imageFile = e.File;
+        var jsImageStream = imageFile.OpenReadStream(1024 * 1024 * 20);
+        var dotnetImageStream = new DotNetStreamReference(jsImageStream);
```
Comment (Keillion) [inline hello-world/blazor/README.md:208]: So here no need the 3 lines.

File: hello-world/blazor/README.md (hunk 1 above)
Comment (felixindynamsoft) [reply hello-world/blazor/README.md]: Agree. thank you

File: hello-world/blazor/README.md (hunk 2 above)
Comment (felixindynamsoft) [reply hello-world/blazor/README.md]: Good catch, thank you!

**Review by Keillion — APPROVED** [review]

**Review by Tom-Dynamsoft — APPROVED** [review]

**Review by eugene-dynamsoft — APPROVED** [review]
LGTM!

**Pull request merged**